docs/photoniq/fingerprinting/get-started-fingerprinting.md

```diff
@@ -0,0 +1,268 @@
+                    })
+                    .catch(e => {
+                        console.log("API call failed");
+                        console.error(e);
+                    });
+            });
+        }
+    </script>
+</header>
+
+<body>
+    <!-- Button to trigger the getVisitorDetails function -->
+    Button: <button onclick="getVisitorDetails()">Get DS Data</button>
+</body>
+</html>
+```
+
+## Fingerprint on Page Load
+
+Perform a fetch at page load time.
+
+```jsx
+<html>
+<head>
+    <script>
+        // Constants
+        const DS_URL = "<URL-TO-DS-SERVICE>";
+       
+        const VISIT_API_ENDPOINT = `${DS_URL}/visits`;
+
+        let fetchPromise;
+
+        // Dynamically load the script when the page loads
+        window.onload = function () {
+            var script = document.createElement('script');
+            script.src = DS_URL;
+            script.onload = function () {
+                // Initialize the agent at startup.
+                var dsPromise = DS_Client.load();
+                window.dsPromise = dsPromise;  // Make it globally accessible
+
+                // Start the fetch request as soon as the DS_Client is loaded.
+                window.dsPromise.then((ds) => ds.get()).then((ds_data) => {
+                    fetchPromise = fetch(VISIT_API_ENDPOINT, {
+                        method: "POST",
+                        headers: {
+                            "Content-Type": "application/json",
+ 
+                        },
+                        body: JSON.stringify(ds_data)
+                    });
+                });
+            };
+            document.body.appendChild(script);
+        };
+
+        // Function to handle visitor details when button is clicked
+        function getVisitorDetails() {
+            if (fetchPromise) {
+                fetchPromise
+                .then(res => {
+                    console.log("API call done!");
+                    console.log("Visitor ID from header:", res.headers.get('x-photoniq-vid'));
+                    return res.json();
+                })
+                .then(data => {
+                    console.log("Visitor ID from response:", data.visitorId);
+                    window.alert('Visitor ID from response: ' + data.visitorId);
+                })
+                .catch(e => {
+                    console.log("API call failed");
+                    console.error(e);
+                });
+            } else {
+                console.log("Data not fetched yet or there was an issue initializing the fetch.");
+            }
+        }
+    </script>
+</head>
+
+<body>
+    <!-- Button to trigger the getVisitorDetails function -->
+    Button: <button onclick="getVisitorDetails()">Get DS Data</button>
+</body>
+
+</html>
+```
+
+### Fingerprint on Text Input
+
+Perform a fetch call when a user starts entering some text in an input field. Here's an example:
```

Comment:
In this example, the visitiorId is fetched when a user starts entering some text in the input field.

I am not sure what is the best way to introduce each example.